x-pack/legacy/plugins/file_upload/public/components/index_settings.js

```diff
@@ -92,11 +80,11 @@ export class IndexSettings extends Component {
       );
     }
 
-    const reg = new RegExp('[\\\\/\*\?\"\<\>\|\\s\,\#]+');
+    const reg = new RegExp('[\\\\/*?"<>|\\s,#]+');
```

Comment:
Why was the regex modified? It might be fine, but I pulled this directly from ML's code since it had been tested and relatively proven

Reply:
Ha.. not sure, that must have been a prettier auto-fix. I'll revert.

Reply:
I'm guessing you just don't need that many "or"s

Reply:
Ok, I've reverted the prettier auto-fixes.